src/SFA.DAS.ApprenticePortal.Web/Pages/_HomeNotifications.cshtml

```diff
@@ -31,3 +32,20 @@
         <p class="govuk-body">You now need to <a asp-external-section="ConfirmMyApprenticeship">confirm your apprenticeship details</a></p>
     </div>
 </div>
+
+@if (Model?.DisplayJustStoppedInfoMessage == true)
```

Comment:
Do we have the `asp-show` tag helper in this project?  I think that may make the markup a touch more expressive.

Reply:
good idea

Reply:
added the tag helper to this project